libraries/cmake/source/librdkafka/CMakeLists.txt

```diff
@@ -141,6 +174,12 @@ function(generateLibrdkafkaxx)
     "${CMAKE_CURRENT_SOURCE_DIR}/config/dummy"
     "${library_root}"
   )
+  if(DEFINED PLATFORM_WINDOWS)
+    target_compile_definitions(thirdparty_librdkafka_cxx PUBLIC
+    LIBRDKAFKA_STATICLIB
```

Comment:
Is this something that should always be defined on all platforms, since we are always generating static libraries?

Reply:
@alessandrogario  That define was removed , now it hangs on OpenSSL on the pipeline but looks like a strawberry perl pipeline issue to me.

Reply:
The failure is due to the previous step actually, though it seems that the powershell script that the CI creates doesn't stop at the first error and launching the subsequent command clears out the exit status.
I'll fix that.

Reply:
Hi @Smjert 
Definition for all platforms would be good. @alessandrogario I have updated the blocks . All green :D